### ISSUE TITLE
Immersive Engineering Waterwheel Blueprint requires too much Waterwheels

### ISSUE THREAD

**Issue by DanteyPL** [Lythom/capsule#30]
## Problem
After creation of blueprint with IE Kinetic Dynamo and 3 IE Waterwheels, blueprint requires 63 IE Waterwheels instead of 3 Waterwheels.

## Expected Behavior
Blueprint requires 3 IE Waterwheels

## Actual Behavior
Blueprint requires 63 IE Waterwheels

## Steps to Reproduce the Problem

  1. Build structure with IE Kinetic Dynamo and 3 IE Waterwheels
  1. Create Capsule with that structure
  1. Create Blueprint Capsule 
  1. Left Click capsule list 63 missing IE Waterwheels instead 3

## Specifications

  - Version of mod: 3.3.11

**Comment by Lythom** [comment]
Thanks for reporting ! 
Will look into this issue after the 1.15.2 port is ready and fix it for both 1.12 and 1.15 👍

**Comment by Lythom** [comment]
This was an issue with the default whitelist_blueprint configuration. I fixed the bug in the default release (to be published soon).
This is something you can already fix in your pack without updating the mod by changing de `config/capsule/blueprint_whitelist.json` file by replacing
```json
{
    "block": "immersiveengineering:wooden_device1",
    "keepNBT": {
      "dummy": null,
      "facing": null,
      "orientation": null,
      "rotation": null
    }
  },
```
by
```json
{
    "block": "immersiveengineering:wooden_device1",
    "keepNBT": {
      "dummy": null,
      "facing": null,
      "orientation": null,
      "rotation": null,
      "offset": null,
      "prevRotation": null
    }
  },
```

Adding this for people searching information on the topic in the futur: I also took the opportunity to improve the documentation on blueprint whitelists (https://github.com/Lythom/capsule/wiki/Modpack-making#whitelist) by adding a video explaining how I fixed the bug to help any packmaker configuring new tile entities blocks to be handled by capsule (https://www.youtube.com/watch?v=6MpXay8rGFw).

Thanks again @NukerNoah for your help improving capsule!
